src/conn/conn_compact.c

```diff
@@ -114,29 +117,34 @@ __compact_server(void *arg)
             continue;
         }
 
+        /* Make a copy of the key has it can be freed once the cursor is released. */
```

Comment:
Typo: as*

Reply:
Nice catch, thank you! 905d5624c0c318c733c89cf07f993b0dd1f7117f